# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ladspa-caps/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- caps-0.2.3.orig/Amp.cc
+++ caps-0.2.3/Amp.cc
@@ -165,7 +165,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<AmpIII>::setup()
 {
 	UniqueID = 1786;
@@ -299,7 +299,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<AmpIV>::setup()
 {
 	UniqueID = 1794;
--- caps-0.2.3.orig/Cabinet.cc
+++ caps-0.2.3/Cabinet.cc
@@ -161,7 +161,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Cabinet>::setup()
 {
 	UniqueID = 1766;
--- caps-0.2.3.orig/Chorus.cc
+++ caps-0.2.3/Chorus.cc
@@ -135,7 +135,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Chorus>::setup()
 {
 	UniqueID = 1767;
@@ -268,7 +268,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<StereoChorus>::setup()
 {
 	UniqueID = 1768;
--- caps-0.2.3.orig/Click.cc
+++ caps-0.2.3/Click.cc
@@ -125,7 +125,7 @@
 	this->ClickStub::init (fs, click, LENGTH (click));
 }
 
-void
+template <> void
 Descriptor<Click>::setup()
 {
 	UniqueID = 1769;
@@ -170,7 +170,7 @@
 	this->ClickStub::init (fs, money, LENGTH (money));
 }
 
-void
+template <> void
 Descriptor<CEO>::setup()
 {
 	UniqueID = 1770;
--- caps-0.2.3.orig/Clip.cc
+++ caps-0.2.3/Clip.cc
@@ -131,7 +131,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Clip>::setup()
 {
 	UniqueID = 1771;
--- caps-0.2.3.orig/Compress.cc
+++ caps-0.2.3/Compress.cc
@@ -142,7 +142,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Compress>::setup()
 {
 	UniqueID = 1772;
--- caps-0.2.3.orig/Eq.cc
+++ caps-0.2.3/Eq.cc
@@ -152,7 +152,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Eq>::setup()
 {
 	UniqueID = 1773;
--- caps-0.2.3.orig/HRTF.cc
+++ caps-0.2.3/HRTF.cc
@@ -134,7 +134,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<HRTF>::setup()
 {
 	UniqueID = 1787;
--- caps-0.2.3.orig/Lorenz.cc
+++ caps-0.2.3/Lorenz.cc
@@ -98,7 +98,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Lorenz>::setup()
 {
 	UniqueID = 1774;
--- caps-0.2.3.orig/Pan.cc
+++ caps-0.2.3/Pan.cc
@@ -145,7 +145,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Pan>::setup()
 {
 	UniqueID = 1788;
--- caps-0.2.3.orig/Phaser.cc
+++ caps-0.2.3/Phaser.cc
@@ -116,7 +116,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Phaser>::setup()
 {
 	UniqueID = 1775;
--- caps-0.2.3.orig/Preamp.cc
+++ caps-0.2.3/Preamp.cc
@@ -108,7 +108,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<PreampIII>::setup()
 {
 	UniqueID = 1776;
@@ -230,7 +230,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<PreampIV>::setup()
 {
 	UniqueID = 1777;
--- caps-0.2.3.orig/Reverb.cc
+++ caps-0.2.3/Reverb.cc
@@ -184,7 +184,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<JVRev>::setup()
 {
 	UniqueID = 1778;
@@ -389,7 +389,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Plate>::setup()
 {
 	UniqueID = 1779;
@@ -486,7 +486,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Plate2x2>::setup()
 {
 	UniqueID = 1795;
--- caps-0.2.3.orig/Roessler.cc
+++ caps-0.2.3/Roessler.cc
@@ -104,7 +104,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Roessler>::setup()
 {
 	UniqueID = 1780;
--- caps-0.2.3.orig/Sin.cc
+++ caps-0.2.3/Sin.cc
@@ -79,7 +79,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<Sin>::setup()
 {
 	UniqueID = 1781;
--- caps-0.2.3.orig/SweepVF.cc
+++ caps-0.2.3/SweepVF.cc
@@ -146,7 +146,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<SweepVF>::setup()
 {
 	UniqueID = 1782;
--- caps-0.2.3.orig/VCO.cc
+++ caps-0.2.3/VCO.cc
@@ -112,7 +112,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<VCOs>::setup()
 {
 	UniqueID = 1783;
@@ -235,7 +235,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<VCOd>::setup()
 {
 	UniqueID = 1784;
--- caps-0.2.3.orig/White.cc
+++ caps-0.2.3/White.cc
@@ -64,7 +64,7 @@
 	}
 };
 
-void
+template <> void
 Descriptor<White>::setup()
 {
 	UniqueID = 1785;

